src/applications/verify/containers/VerifyApp.jsx

```diff
@@ -51,46 +78,44 @@ export class VerifyApp extends React.Component {
     }
   }
 
-  renderVerifyButton(signInMethod) {
-    const verifyWithLoginGov =
-      this.signinMethodLabels.logingov === signInMethod;
-
-    const renderOpts = {
-      copy: verifyWithLoginGov ? 'Login.gov' : 'ID.me',
-      renderImage: verifyWithLoginGov ? (
-        <LoginGovSVG />
-      ) : (
-        <img
-          role="presentation"
-          aria-hidden="true"
-          alt="ID.me"
-          src="/img/signin/idme-icon-white.svg"
+  verifyButton(signInMethod) {
+    if (
+      signInMethod === this.signinMethodLabels.idme ||
+      signInMethod === this.signinMethodLabels.logingov
+    ) {
+      const selectCSP = signInCSP =>
+        this.renderOpts.find(csp => csp.copy === signInCSP);
+      const { className, copy, renderImage, policy } = selectCSP(signInMethod);
```

Comment:
*Optional - No need to destructure here as we can give it an arbitrary variable name and use an object spread for props into VerifyButton component.

```
const verifyButtonProps = selectCSP(signInMethod);
<VerifyButton {...verifyButtonProps} />
```

Reply:
Makes sense. I've gone ahead and updated the code to reflect this suggestion.